data_steward/validation/participants/store_participant_summary_results.py

```diff
@@ -125,6 +127,48 @@ def fetch_and_store_ps_hpo_data(client,
     LOGGER.info(f'Done.')
 
 
+def fetch_and_store_full_ps_data(client,
+                                 project_id,
+                                 rdr_project_id,
+                                 dataset_id=DRC_OPS):
+    """
+    Fetches PS API data for all participants and stores in drc_ops.ps_api_unioned table
+
+    :param client: BQ client
+    :param project_id: Identifies the project
+    :param rdr_project_id: PS API project
+    :param dataset_id: contains table to store PS API data
+    :return: 
+    """
+
+    # Get participant summary data
+    LOGGER.info(f'Getting participant summary data')
+    participant_info = get_all_participant_information(rdr_project_id)
+
+    # Load schema
+    schema = bq.get_table_schema(PS_API_VALUES)
+    # TODO use resources.get_table_id after updating it to flip hpo_id, table_name
+    table_name = f'{PS_API_VALUES}_{UNIONED}'
```

Comment:
Since we are getting rid of all the site-specific ps_api tables. it is good to name this table just `ps_api_values`